src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -79,6 +80,17 @@
    jruby-request/wrap-with-error-handling
    ring/wrap-params))
 
+(defn static-file-content-request-handler
+  "Build the main request handler for the /static_file_content endpoint"
+  [get-code-content]
+  (fn [req]
+    (let [environment (get-in req [:params "environment"])
+          code-id (get-in req [:params "code-id"])
+          file-path (get-in req [:params :rest])
```

Comment:
Oh, I see what's going on now.  I thought that `file-path` was a query parameter like the others, which is incorrect, so this is probably good as-is.